modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/PodcastDao.kt

```diff
@@ -354,13 +354,14 @@ abstract class PodcastDao {
 
     @Query(
         """
-        SELECT DISTINCT podcast_episodes.uuid as episodeId, podcasts.uuid, podcasts.title, podcasts.author, podcasts.primary_color as tintColorForLightBg, podcasts.secondary_color as tintColorForDarkBg, SUM(podcast_episodes.played_up_to) as totalPlayedTime, COUNT(podcast_episodes.uuid) as numberOfPlayedEpisodes
-        FROM podcast_episodes
-        JOIN podcasts ON podcast_episodes.podcast_id = podcasts.uuid
-        WHERE podcast_episodes.last_playback_interaction_date IS NOT NULL AND podcast_episodes.last_playback_interaction_date > :fromEpochMs AND podcast_episodes.last_playback_interaction_date < :toEpochMs
-        GROUP BY podcast_id
-        ORDER BY totalPlayedTime DESC, numberOfPlayedEpisodes DESC
-        LIMIT :limit
+         SELECT (totalPlayedTime * 0.4) * (numberOfPlayedEpisodes * 0.6) as weighted, * FROM (
```

Comment:
This matches iOS query: https://github.com/Automattic/pocket-casts-ios/pull/1192/commits/844bd08715e7a6d51466448e6283947c94e3872e for top podcast

Slack discussion: p1698773178757819-slack-C041BGTFQ5R